etl/steps/data/garden/war/2024-01-11/nuclear_threat_initiative_overview.meta.yml

```diff
@@ -0,0 +1,16 @@
+dataset:
+  update_period_days: 365
+
+tables:
+  nuclear_threat_initiative_overview:
+    variables:
+      status:
+        title: Nuclear weapons status
+        description_short: Whether a country does not consider (0), considers (1), pursues (2), or possesses (3) nuclear weapons.
+        processing_level: major
```

Comment:
Again, it would be cool to add a `description_processing` since the processing level is major

Reply:
Done.